src/docs/sdk/performance/span-data-conventions.mdx

```diff
@@ -26,12 +26,12 @@ Below describes the conventions for the Span interface for the `data` field on t
 
 ## Browser
 
-| Attribute                          | Type   | Description                                 | Examples              |
-| ---------------------------------- | ------ | ------------------------------------------- | --------------------- |
-| `url`                              | string | The URL of the resource that was fetched.   | `https://example.com` |
-| `method`                           | string | The HTTP method used.                       | `GET`                 |
-| `type`                             | string | The type of the resource that was fetched.  | `xhr`                 |
-| `Encoded Body Size`                | number | The encoded body size of the request.       | `123`                 |
-| `Decoded Body Size`                | number | The decoded body size of the request.       | `456`                 |
-| `Transfer Size`                    | number | The transfer size of the request.           | `789`                 |
-| `resource.render_blocking_status`  | string | The render blocking status of the resource. | `non-blocking`        |
+| Attribute                              | Type   | Description                                 | Examples              |
+| -------------------------------------- | ------ | ------------------------------------------- | --------------------- |
+| `url`                                  | string | The URL of the resource that was fetched.   | `https://example.com` |
+| `method`                               | string | The HTTP method used.                       | `GET`                 |
+| `type`                                 | string | The type of the resource that was fetched.  | `xhr`                 |
+| `http.encoded_response_content_length` | number | The encoded body size of the request.       | `123`                 |
```

Comment:
I think we can actually just use `http.X` for everything here - since it is just http information.

To match otel I think we use `http.request_content_length` to represent the encoded body size, and introduce `http.decoded_request_content_length` to represent the decoded body size.

We should still indicate that what these fields used to be called for reference (since older SDKs will keep sending this info).

Reply:
Okay, I've called out the original names in the descriptions and used `http.request_content_length`, `http.decoded_request_body_length`, and `http.transfer_size`.